src/go/goblins/docker/docker.go

```diff
@@ -0,0 +1,209 @@
+//  Copyright 2018 Google LLC
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+
+//        https://www.apache.org/licenses/LICENSE-2.0
+
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//	limitations under the License.
+
+// Package docker contains libraries to access and handle docker containers.
+// This feature will *only* work on Linux OS with appropriate Kernel support.
+package docker
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+const aufsPath = "/sbin/mount.aufs"
+
+// Mount mounts the container's filesystem into a mount directory.
+// Note that this method just executes a bunch of shell command
+// to build the filesystem. Not much magic here.
+// All of this is based on https://github.com/google/docker-explorer/blob/master/docker_explorer/lib/storage.py
```

Comment:
Why not use docker libs?

Reply:
So, I've tried to understand how the docker libs work.
Man is that a tangled ununderstandable bit of code. My go readability is poor of course, so it might just be me. I'll happily take pointers and pop the champagne for something that uses docker directly.